### DISCUSSION TITLE
iBlinds V2: "specific" type is incorrect for 2 of 6 devices...

### DISCUSSION THREAD

**Discussion by danielbrunt57** [zwave-js/zwave-js#1683] [Q&A] [answered]
I have 6 iBlinds V2 Window Blind Controller motors
1 is still attached to HomeSeer but 5 are now in HA.
Of those 5, three report:
```
  "deviceClass": {
    "basic": 4,
    "generic": 17,
    "specific": 6
```
but two of them report:
```
  "deviceClass": {
    "basic": 4,
    "generic": 17,
    "specific": 0
```
and end up with the multilevel switch being a `light` rather than `cover`
The firmware on all of them is identical.

**Any suggestions on how I might override this "specific": 0 to be 6???**

17:10 PST: I've just moved my 6th motor from HomeSeer to HA ZWaveJS and it added as a cover.  There would appear to be an issue with firmware settings on 2/6 motors reporting "specific": 0, that or not reporting and ZWJS defaulting to 0.

**Comment by AlCalzone** [comment, marked as answer]
These values are retrieved in one command, so we're not defaulting to anything. I'd start by asking the manufacturer for support - something is wrong with those two.
We currently don't have any flags to override these values.

**Reply by danielbrunt57** [reply]
Thanks.
